downloader/src/main.rs

```diff
@@ -20,6 +20,9 @@ async fn main() -> Result<()> {
         .nth(1)
         .context("provide url to rss as first argument")?;
 
+    /// TODO: There are some situation when `args()` will return all arguments only once and will
```

Comment:
Yeah for now it will do. Once the interface is stable I will definitely use clap. I'm not building on nix - just using it to provide os dependencies using direnv.

I've tried to build the whole app using crane and nix flake but after a night of banging my head I gave up.
Would love to give it a go again, though.